experimental/devices/sn3218/sn3218.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package sn3218
+
+import (
+	"errors"
+
+	"periph.io/x/periph/conn/i2c"
+)
+
+const (
+	i2cAddress             = 0x54
+	cmdEnableOutput        = 0x00
+	cmdSetBrightnessValues = 0x01
+	cmdEnableLeds          = 0x13
+	cmdUpdate              = 0x16
+	cmdReset               = 0x17
+)
+
+type dev struct {
+	i2c        *i2c.Dev
+	states     [18]bool
+	brightness [18]byte
+}
+
+// New returns an object that handles communication with a SN3218 LED driver
```

Comment:
// New returns an handle to a SN3218 LED driver.

Reply:
done